signer/src/testing/dummy.rs

```diff
@@ -179,16 +179,17 @@ impl Dummy<bitcoin::TxIn> for BitcoinTxVin {
 
 impl Dummy<Faker> for BitcoinBlockInfo {
     fn dummy_with_rng<R: Rng + ?Sized>(_: &Faker, rng: &mut R) -> Self {
-        let block_height = (0..i64::MAX as u64).choose(rng).unwrap();
-        BitcoinBlockInfo::random_with_height(block_height, rng)
+        let height = (0..i64::MAX as u64).choose(rng).unwrap();
```

Comment:
I think you can do something like `let height: BitcoinBlockHeight = faker.fake_with_rng(&mut rng);`

Reply:
Oh right, will update.